apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/modal.js

```diff
@@ -11,6 +11,8 @@ import { useCanvas } from './use-canvas';
 
 import './modal.scss';
 
+const GLOBAL_STYLES_TREATMENT_GROUP = '1';
```

Comment:
Since this is also used by `apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/notices.js` we could add the constant to a common file and include it instead of re-declaring it.

Reply:
I don't see any good common file where I could add it, so rather than creating a new file I'll keep the duplicated constant here. It's temporary code at the end we're going to get rid of eventually.